ktor-server/ktor-server-cio/jvm/src/io/ktor/server/cio/CIOApplicationEngine.kt

```diff
@@ -54,7 +54,7 @@ public class CIOApplicationEngine(environment: ApplicationEngineEnvironment, con
         try {
             environment.connectors.forEach { connectorSpec ->
                 if (connectorSpec.type == ConnectorType.HTTPS) {
-                    throw UnsupportedOperationException("HTTPS is not supported by CIO engine")
+                    throw UnsupportedOperationException("CIO Engine does not currently support HTTPS. Please consider using a different engine if you require HTTPS")
```

Comment:
> Please consider using a different engine if you require HTTPS

Isn't that implied from the first sentence?

Reply:
It's providing a helpful step as to what to do.